serial_only_wallet/serial_only_wallet.ino

```
@@ -171,6 +167,10 @@ void setup() {
     // serial connection
     Serial.begin(9600);
     // loading master private key
+    if (!SD.begin(4)){
+        Serial.println("error: no SD card");
+        return;
```

I'm not sure whether this is right. Should we raise an error instead?